src/linux/cgroups2.cpp

```diff
@@ -259,12 +265,11 @@ Try<Nothing> create(const string& cgroup, bool recursive)
 
 Try<Nothing> destroy(const string& cgroup)
 {
-  const string absolutePath = path::join(MOUNT_POINT, cgroup);
-
-  if (!os::exists(absolutePath)) {
-    return Error("There does not exist a cgroup at '" + absolutePath + "'");
+  if (!cgroups2::exists(cgroup)) {
+    return Error("Cgroup '" + cgroup + "' does not exist");
   }
 
+  const string absolutePath = path::join(MOUNT_POINT, cgroup);
```

Comment:
Can remove this change, but I think it's relevant because we're replacing the if statement check if an equivalent but more readable check.